fcli-other/fcli-functional-test/src/ftest/groovy/com/fortify/cli/ftest/ssc/SSCReportTemplateSpec.groovy

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text 
+ * and its affiliates and licensors ("Open Text") are as may 
+ * be set forth in the express warranty statements accompanying 
+ * such products and services. Nothing herein should be construed 
+ * as constituting an additional warranty. Open Text shall not be 
+ * liable for technical or editorial errors or omissions contained 
+ * herein. The information contained herein is subject to change 
+ * without notice.
+ */
+package com.fortify.cli.ftest.ssc
+
+import static com.fortify.cli.ftest._common.spec.FcliSessionType.SSC
+
+import com.fortify.cli.ftest._common.Fcli
+import com.fortify.cli.ftest._common.spec.FcliBaseSpec
+import com.fortify.cli.ftest._common.spec.FcliSession
+import com.fortify.cli.ftest._common.spec.Prefix
+import com.fortify.cli.ftest.ssc._common.SSCAppVersion
+
+import spock.lang.AutoCleanup
+import spock.lang.Requires
+import spock.lang.Shared
+
+@Prefix("ssc.report-template") @FcliSession(SSC) 
+class SSCReportTemplateSpec extends FcliBaseSpec {
+    
+    def "list"() {
+        def args = "ssc report-template list"
+        when:
+            def result = Fcli.run(args)
+        then:
+            verifyAll(result.stdout) {
+                size()>0
+                it[0].replace(' ', '').equals("IdNameTypeTemplatedocidInuse")
+                it.any { it.startsWith(" 1") }
+            }
+    }
+    
+    //TODO add tests for create, get, download, generate-config, delete
```

Comment:
For get/download/..., either assume standards report bundle has been seeded, or first use the `create` command to install some dummy fcli-test report definition, such that the other commands can assume that this particular definition exists.